src/auth/cognito/types/options/index.ts

```diff
@@ -0,0 +1,13 @@
+export {AutoSignInOptions} from "./autoSignInOptions"
```

Comment:
For the purposes of the RFC, I wonder if it would make sense to merge all of the option types into a single file to simplify review for customers? Regardless of what we do for the final implementation. Thoughts? (same argument could be applied to requests, results, steps, etc.)

Reply:
yes, that's a good feedback. I just updated it. Now there are 15 files instead of 91